src/TropicalGeometry/numbers.jl

```diff
@@ -0,0 +1,449 @@
+# module Tropical
+
+export tropical_numbers,
+       @tropical,
+       determinant,
+       permanent
+
+# using Reexport
+
+# @reexport using Oscar
+
+# import Oscar: expressify, Ring, @enable_all_show_via_expressify, zero, one, iszero, isone, PolynomialRing
+
+################################################################################
+#
+#  Types
+#
+################################################################################
+
+# We use T to record whether we are in the min/max case
+# T is either typeof(min) or typeof(max)
+mutable struct TropicalNumbers{T} <: Field
```

Comment:
As above, this could be turned into a singleton type.
```suggestion
struct TropicalNumbers{T} <: Field
```

Reply:
Alas, isn't this and `TropicalSemiring` the same thing, just with a different name?!

Reply:
Apparently `numbers.jl` was just a left-over and not needed anymore. It has been removed.